src/Translator.js

```diff
@@ -2,6 +2,7 @@ import baseLogger from './logger.js';
 import EventEmitter from './EventEmitter.js';
 import postProcessor from './postProcessor.js';
 import * as utils from './utils.js';
+import { isOptional } from './utils.js';
```

Comment:
does not need an additional import -> can be accessed as utils.isOptional

personally I wouldn't add a helper in utils for that but have those written inline:

translate -> `if (keys === undefined || keys === null) return '';`
exists -> `if (keys === undefined || keys === null) return false;`


having a reusable helper function makes no sense -> it's not usable in other places, isOptional is non-descriptive if not knowing where it is used...

Reply:
Yes, I'll change it to inline.